src/test/java/com/books/BookControllerTest.java

```diff
@@ -45,7 +45,7 @@ public void getAllBooks() throws Exception {
         allBooks.add(new Book("Book Title", "Book Author", "Book Summary 22", 2 ));
         allBooks.add(new Book("Book Title Second", "Book Author Second", "Book Summary 23 Second", 2 ));
 
-        when(bookService.getAllBooks()).thenReturn(allBooks);
+        when(bookService.getAllBooks(0,20)).thenReturn(allBooks);
```

Comment:
just for info: if the value does not matter you can use any() - https://stackoverflow.com/questions/1778744/using-mockitos-generic-any-method

Reply:
Done using anyInt()